Makefile

```diff
@@ -167,13 +190,25 @@ OBJS := \
 OBJS := $(addprefix $(OUT)/, $(OBJS))
 deps := $(OBJS:%.o=%.o.d)
 
+EXPORTED_FUNCS := _main
+EMCC_CFLAGS :=
+EMCC_MACRO :=
+ifeq ("$(CC_IS_EMCC)", "1")
+EMCC_MACRO += -DMEM_SIZE=0x40000000    # 1GB
```

Comment:
The name `EMCC_MACRO` is misleading. Drop it and move to `CFLAGS_emcc`.

Reply:
Instead of "1GB," write "1 GiB" in [binary prefix](https://en.wikipedia.org/wiki/Binary_prefix) form.